x-pack/plugins/profiling/public/components/no_data_page.tsx

```diff
@@ -265,43 +267,100 @@ docker.elastic.co/observability/profiling-agent:${hostAgentVersion} /root/pf-hos
       steps: [
         {
           title: i18n.translate('xpack.profiling.tabs.symbols.step1', {
-            defaultMessage: 'Download the symbtool binary from the URL below:',
+            defaultMessage: 'Download the symbtool binary',
           }),
           content: (
-            <EuiLink target="_blank" href={`https://ela.st/symbtool-linux-amd64`}>
-              https://ela.st/symbtool-linux-amd64
-            </EuiLink>
+            <EuiText>
+              <EuiLink target="_blank" href={`https://ela.st/symbtool-linux-amd64`}>
+                {i18n.translate('xpack.profiling.tabs.symbols.step1.amd64', {
+                  defaultMessage: 'Linux x86_64 / AMD64 executable',
+                })}
+              </EuiLink>
+              <EuiSpacer size="s" />
+              <EuiLink target="_blank" href={`https://ela.st/symbtool-linux-arm64`}>
+                {i18n.translate('xpack.profiling.tabs.symbols.step1.arm64', {
+                  defaultMessage: 'Linux aarch64 / ARM64 executable',
+                })}
+              </EuiLink>
+            </EuiText>
           ),
         },
         {
           title: i18n.translate('xpack.profiling.tabs.symbols.step2', {
-            defaultMessage: 'Generate an Elasticsearch token:',
+            defaultMessage: 'Generate an Elasticsearch token',
           }),
           content: (
-            <EuiLink
-              target="_blank"
-              href={`https://www.elastic.co/guide/en/kibana/master/api-keys.html`}
-            >
-              {i18n.translate('xpack.profiling.tabs.symbols.step2.instructions', {
-                defaultMessage: 'Instructions here',
-              })}
-            </EuiLink>
+            <EuiText>
+              <EuiLink
+                target="_blank"
+                href={`https://www.elastic.co/guide/en/kibana/master/api-keys.html`}
+              >
+                {i18n.translate('xpack.profiling.tabs.symbols.step2.instructions', {
+                  defaultMessage: 'Instructions here',
+                })}
+              </EuiLink>
+            </EuiText>
```

Comment:
Added the `EuiText` wrapper around here to achieve uniform typography: without this, the links are rendered with a noticeably smaller font.